Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -773,11 +773,10 @@ public static void DisableDevice(InputDevice device)
             s_Manager.EnableOrDisableDevice(device, false);
         }
 
-        ////REVIEW: should this be a device-level reset along with sending the reset IOCTL
-        ////        or a control-level reset on just the memory state?
-        public static void ResetDevice(InputDevice device)
+        public static unsafe void ResetDevice(InputDevice device)
         {
-            throw new NotImplementedException();
+            var resetCommand = RequestResetCommand.Create();
```

Comment:
Null-check device and throw `ArgumentNullException` if necessary.

Reply:
I do